src/thrift.js

```diff
@@ -11,15 +11,16 @@
 // or implied. See the License for the specific language governing permissions and limitations under
 // the License.
 
-import fs from 'fs';
 import opentracing from 'opentracing';
 import path from 'path';
 import { Thrift } from 'thriftrw';
 import Utils from './util.js';
+import ThriftData from './generated/thrift';
 
 export default class ThriftUtils {
   static _thrift = new Thrift({
```

Comment:
Seems like we're loading the same Thrift IDLs over and over, perhaps it's worth extracting everything into a single module (although not strictly in scope of this PR).

Reply:
Agree, and probably should be another PR to avoid conflating the refactor with these changes.

Reply:
Up to you. I would do it in this PR, since you're changing all the call sites already, so rather than passing file paths, you could simply import a symbol from a helper module.

Reply:
That's a fair point. I'll give it a shot.